src/main/java/io/confluent/idesidecar/restapi/credentials/Password.java

```diff
@@ -13,6 +14,7 @@
  * returned in an API response.
  */
 @JsonDeserialize(using = Password.Deserializer.class)
+@JsonSerialize(using = Password.Serializer.class)
```

Comment:
My concern with doing this is that this serializer is used to serialize the `Redactable` objects _in the responses_. We absolutely cannot allow that.

We might need to not register this serializer, and *only register the serializer in the `SidecarClient`'s `ObjectMapper` instance*, something akin to:

```
ObjectMapper mapper = new ObjectMapper();
SimpleModule module = new SimpleModule();
module.addSerializer(Password.class, new Password.Serializer());
module.addSerializer(ApiSecret.class, new ApiSecret.Serializer());
mapper.registerModule(module);
```